### PR TITLE
feat(services): apply action to more than one service deploy stop etc

### PR DESCRIPTION
**Pull request by ctjhoa**
# What does this PR do?

https://qovery.atlassian.net/browse/FRT-687

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28102cd2-684e-4ecb-a4f7-2f37f653b027 | [my-helm-env](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/28102cd2-684e-4ecb-a4f7-2f37f653b027/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> One day they will all understand

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [\[PR\] my-env - feat(services): apply action to more than one service deploy stop etc - 2024-02-07T13:52:22Z](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/e773b890-e6b2-4406-93a9-71f9a7ab6d73/environment/cec4a589-0dbf-458f-bfea-b4a56418465e/services/general)

 Another comment will be posted when deployments are terminated

**Comment by acarranoqovery** [comment]
[Qovery Preview](https://www.qovery.com)
---

Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://1163-console-staging.zc531a994.rustrocks.cloud)

File: libs/domains/services/feature/src/lib/service-list/service-list-action-bar.tsx [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    openModalConfirmation({
+      title: `Delete ${deletableServices.length} ${pluralize(deletableServices.length, 'service')}`,
+      name: 'these services',
+      isDelete: true,
+      action: async () => {
+        try {
+          await deleteAllServices({
+            environment,
+            payload: {
+              application_ids: deletableServices
```
Comment (RemiBonnet) [inline libs/domains/services/feature/src/lib/service-list/service-list-action-bar.tsx:232]: could you create a function with to manage the payload? I see you use this several times, I think that would simplify the code

**Review by RemiBonnet — CHANGES_REQUESTED** [review]
Thanks @ctjhoa !

I added some comments + I see small improvements we can done to be iso with Figma:

- I think you don't have a good font size for "clear selection" should be `14px`
- The height of the toolbar should be `52px`
- The spacing between element isn't correct 
- The spacing between label and icon for the btn "More"

**Integration:**
Capture d’écran 2024-02-13 à 08 46 47

**Figma:**
Capture d’écran 2024-02-13 à 08 47 32

----

I think the global checkbox has an shift of 1px when is selected 
https://www.loom.com/share/e17a7a9a7ad44a45849b25523cd02bac

**Review by RemiBonnet — APPROVED** [review]

**Pull request merged**